addons/nuxeo-easyshare/nuxeo-easyshare-web-ui/src/main/resources/web/nuxeo.war/ui/document/easysharefolder/nuxeo-easysharefolder-view-layout.html

```diff
@@ -45,9 +45,7 @@
                            empty-label="[[i18n('collections.noDocument')]]"
                            empty-label-when-filtered="[[i18n('collections.noDocumentWhenFiltered')]]"
                            hide-content-view-actions>
-      <div slot="selectionActions">
-        <nuxeo-slot slot="COLLECTION_ACTIONS" model="[[actionContext]]"></nuxeo-slot>
-      </div>
+      <nuxeo-slot slot="COLLECTION_ACTIONS" model="[[actionContext]]"></nuxeo-slot>
```

Comment:
`selectionActions` is still the target native slot here, but after ELEMENTS-1012 we're not using the `name` attribute to define the `nuxeo-slot`'s name:
```suggestion
      <nuxeo-slot slot="COLLECTION_ACTIONS" slot="selectionActions" model="[[actionContext]]"></nuxeo-slot>
```